virt-v2v/cold/entrypoint.go

```diff
@@ -73,23 +101,31 @@ func main() {
 	}
 
 	if source == vSphere {
-		if _, err := os.Stat("/etc/secret/cacert"); err == nil {
-			// use the specified certificate
-			err = os.Symlink("/etc/secret/cacert", "/opt/ca-bundle.crt")
-			if err != nil {
-				fmt.Println("Error creating ca cert link ", err)
-				os.Exit(1)
-			}
-		} else {
-			// otherwise, keep system pool certificates
-			err := os.Symlink("/etc/pki/tls/certs/ca-bundle.crt.bak", "/opt/ca-bundle.crt")
-			if err != nil {
-				fmt.Println("Error creating ca cert link ", err)
-				os.Exit(1)
+		if os.Getenv("V2V_test") != "true" {
```

Comment:
ah?..

Reply:
yeah.. in the tests we can't do the symbol linking.. i tried to overcome it in a creative way

i'm up for suggestions

Reply:
maybe just pulling it out of this function will do :)

Reply:
> maybe just pulling it out of this function will do :)

yep, that's a better approach - it was an indication of having too many responsibilities within the same function